tests/outline/outline2.cc

```diff
@@ -112,10 +100,9 @@ c:@F@LoadCompilationEntriesFromDirectory#&1$@N@std@N@__cxx11@S@basic_string>#C#$
       "id": 2,
       "usr": "c:@S@CompilationEntry@FI@args",
       "short_name": "args",
-      "detailed_name": "std::vector<std::string> CompilationEntry::args",
+      "detailed_name": "int CompilationEntry::args",
```

Comment:
I don't understand why it is `int` in `--test-index`. But when I copy the file out and index it with `--language-server`, it is good.

Reply:
It is failing to resolve system headers. This should probably be fixed before merging.

Reply:
`--test-index` can find system headers and parse them. That's why `std::string` `std::vector` are recognized when used along. However, their combination `std::vector<std::string>` is not recognized as `int` in `--test-index` but this works fine in standalone mode.